build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-app="oracle-image"
+app="crypto-oracle"
 docker build -t ${app} .
```

Comment:
I think it is better if we can have version tagging every time we build. Usually, we use `git describe` to produce unique version tag for us. In case you want to have that, I can show you how the other teams are doing.